api/swagger.yaml

```diff
@@ -8263,3 +8263,62 @@ paths:
           format: "int64"
           required: true
       tags: ["Secret"]
+  /distribution/{name}/json:
```

Comment:
Yes, that issue came up during a meeting earlier this week. Especially colons `:` in URL's can be troublesome (using a `?something=` query-string is better)

it's consistent with all endpoints we have now though, so not sure we should make the exception here (open to input on that one)